src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysis.DisposeDataFlowOperationVisitor.cs

```diff
@@ -175,7 +176,8 @@ protected override void EscapeValueForParameterPointsToLocationOnExit(IParameter
             {
                 Debug.Assert(!escapedLocations.IsEmpty);
                 Debug.Assert(parameter.RefKind != RefKind.None);
-                var escapedDisposableLocations = escapedLocations.Where(l => l.LocationTypeOpt?.IsDisposable(WellKnownTypeProvider.IDisposable) == true);
+                var escapedDisposableLocations =
+                    escapedLocations.Where(l => l.LocationTypeOpt?.IsDisposable(WellKnownTypeProvider.GetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemIDisposable)) == true);
```

Comment:
Can you extract out `WellKnownTypeProvider.GetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemIDisposable)` into a local?

Reply:
You won't need to extract to a local if you implemented the suggestion https://github.com/dotnet/roslyn-analyzers/pull/2871/files#r331985326